worker/instancemutater/worker_test.go

```diff
@@ -459,6 +496,98 @@ func logIt(c *gc.C, level loggo.Level, message string, args interface{}) {
 	c.Logf("%s "+message, nArgs...)
 }
 
+type workerContainerSuite struct {
+	workerSuite
+
+	containerTag names.Tag
+	container    *mocks.MockMutaterMachine
+}
+
+var _ = gc.Suite(&workerContainerSuite{})
+
+func (s *workerContainerSuite) SetUpTest(c *gc.C) {
+	s.workerSuite.SetUpTest(c)
+
+	s.containerTag = names.NewMachineTag("0/lxd/0")
+	s.newWorkerFunc = instancemutater.NewContainerWorker
+}
+
+// TestFullWorkflow uses the the expectation scenarios from each of the tests
+// below to compose a test of the whole instance mutator scenario, from start
+// to finish for a ContainerWorker.
+func (s *workerContainerSuite) TestFullWorkflow(c *gc.C) {
```

Comment:
Given the overlap between the Environ and Container workers, testing the full workflow of the container worker should be sufficient.